sql/src/main/java/io/crate/executor/transport/AlterTableOperation.java

```diff
@@ -380,6 +394,13 @@ public AlterTableOperation(ClusterService clusterService,
         return userManager.transferTablePrivileges(sourceIdent, targetIdent);
     }
 
+    private CompletableFuture<Long> transferIngestRules(String sourceIdent, String targetIdent) {
+        FutureActionListener<TransferIngestRuleResponse, Long> listener =
+            new FutureActionListener<>(TransferIngestRuleResponse::affectedRows);
```

Comment:
`testRenameClosedTable` test is failing now because it expects -1L, maybe cause of this?